cterasdk/edge/config.py

```diff
@@ -18,6 +18,7 @@ def set_location(self, location):
         """
         Set the location of the gateway
 
+        :param str hostname: New location to set
         :return str: The command response
```

Comment:
The set location command returns the location that was set (as a string)

Reply:
Fixed